zcash_client_backend/CHANGELOG.md

```diff
@@ -69,6 +82,8 @@ and this library adheres to Rust's notion of
 ### Removed
 - `zcash_client_backend::PoolType::is_receiver`: use
   `zcash_keys::Address::has_receiver` instead.
+- `zcash_client_backend::DecryptedTransaction::sapling_outputs` use
+  the `DecryptedTransaction::outputs` method instead.
```

Comment:
```suggestion
- `zcash_client_backend::DecryptedTransaction::sapling_outputs` (use
  the `DecryptedTransaction::outputs` method instead).
```

Reply:
I restored the `sapling_outputs` method; it ultimately didn't make sense to unify the output vectors at this level.